src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -47,7 +47,7 @@ uint8_t POWERMGNT::powerToCrsfPower(PowerLevels_e Power)
     case PWR_500mW: return 4;
     case PWR_1000mW: return 5;
     case PWR_2000mW: return 6;
-    case PWR_50mW:
+    case PWR_50mW: return 8;
```

Comment:
Minor, but do we want to order this by power level or returned value?

Reply:
Imo this makes it more clear that the power levels are not in order.

Reply:
@JyeSmith Do you want a new commit for this or should I amend it to the previous one? Some projects are picky about that and want to keep a clean commit history, others don't care.

Reply:
I'm not picky, either is fine.

Reply:
Yeah we squash commit our PRs so even a new commit will get squashed down to 1